apple/job/apple.py

```diff
@@ -23,8 +23,8 @@ class AppleCookies:
 
 # Add a cache decorator to the function that fetches the cookies
 # so that we don't have to fetch them every time.
-# The cache will be invalidated after 10 minutes.
-@filecache(10 * 60)
```

Comment:
maybe we should increase this? especially as we also pay for every single login. e.g. 30min?